release/pkg/versions.go

```diff
@@ -98,7 +100,10 @@ func (v *cliVersioner) patchVersion() (string, error) {
 	return v.cliVersion, nil
 }
 
-func generateComponentHash(hashes []string) string {
+func generateComponentHash(hashes []string, dryRun bool) string {
```

Comment:
Related to this: https://github.com/aws/eks-anywhere/pull/1743#discussion_r840728120

Reply:
@g-gaston @abhay-krishna I'm not thrilled about leaving this as a follow up. It feels like we're just churning out tech debt. This seems like something we should address now.

Reply:
So the only reason we need the bool in the function is to return a constant fake hash value in case we're doing a dry run and not an actual release. Do you think we can have a getter method for the constant and do the if check bifurcation in these assets files instead of in the method to compute manifests hash?

Reply:
I have addressed this using the method described above.